### PR TITLE
[#IOPAE-1137] Add GetFeaturedServices and GetFeaturedInstitutions fn

### PR DESCRIPTION
**Pull request by StefanoDoc**
Change `/featured` API into 2 new api `/institutions/featured` and `/services/featured`
#### List of Changes

- add new GET `/institutions/featured`
- add new GET `/services/featured`
- remove old GET `/featured`
- Update config and infra

#### How Has This Been Tested?
unit tests

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by giuseppedipinto — CHANGES_REQUESTED** [review]

File: infra/prod/italynorth/main.tf
Comment (StefanoDoc) [reply infra/prod/italynorth/main.tf]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: infra/prod/italynorth/main.tf
Comment (StefanoDoc) [reply infra/prod/italynorth/main.tf]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: infra/_modules/function_app/variables.tf
Comment (StefanoDoc) [reply infra/_modules/function_app/variables.tf]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: infra/_modules/function_app/variables.tf
Comment (StefanoDoc) [reply infra/_modules/function_app/variables.tf]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: apps/app-backend/src/config.ts
Comment (StefanoDoc) [reply apps/app-backend/src/config.ts]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

File: apps/app-backend/src/config.ts
Comment (StefanoDoc) [reply apps/app-backend/src/config.ts]: [c9ae89d](https://github.com/pagopa/io-services-cms/pull/841/commits/c9ae89da5349cf7765188ac39cc7e0359a9f9c95)

**Review by giamma1295 — COMMENTED** [review]
LGTM but wait for checks on infra itn action failure before merge!

**Review by giuseppedipinto — APPROVED** [review]

**Pull request merged**
